exporter/prometheus/prometheus_test.go

```diff
@@ -15,12 +15,17 @@
 package prometheus
 
 import (
+	"context"
```

Comment:
"golang.org/x/net/context" and please move it an import a line after the first stdlib import.

Reply:
Why not use the context in the stdlib?

Reply:
@odeke-em ^

Reply:
@freddygv we still need to support gRPC which uses go1.6,go1.7 which use "golang.org/x/net/context"

Reply:
@odeke-em we support 1.8+ only, we use "context" in a number of places so I think it's fine

Reply:
Oh cool, thank you @ramonza for the update. I'll also page @rakyll to ask if this intentional.